src/changelog/databases/_posts/2024-10-01-postgresql-14.13.0-2.md

```diff
@@ -0,0 +1,18 @@
+---
+modified_at: 2024-10-01 09:00:00
+title: 'PostgreSQL - 14.13.0-2, 13.16.0-2 and 12.20'
+---
+
+Changelog:
+
+- New Scalingo patch versions available:
+  - v14.13.0-2
+  - v13.16.0-2
+- New minor version available:
+  - v12.20
```

Comment:
In `database-api-service`, looking at https://github.com/Scalingo/database-api-service/blob/master/db/seeds.rb#L43
the versions are slightly different: 14.13.0-1, 13.16.0-1, 12.18.0-1. Where is the source of truth ? And should not we have the same versions everywhere ?

Reply:
Our internal procedure does not mention updating the seeds. That's why I forgot to update the seeds. I'll update our internal procedure.